docs/technical/tests/frontend/components.rst

```diff
@@ -0,0 +1,37 @@
+Component Tests
+===============
+
+When it comes to testing React components, we need a way to test components in insolation without needing to mount the entire application. Many of our current React components are tested with `Enzyme <https://airbnb.io/enzyme/>`_, but more recently, our tests have started to use `React Testing Library <https://testing-library.com/docs/react-testing-library/intro>`_ instead. (See `this blog post by Kent Dodds for more information <https://kentcdodds.com/blog/introducing-the-react-testing-library>`_.)
+
+The main reason to adopt React Testing Library was to avoid testing too much implementation details, especially for React components. You want your tests to be maintainable and don't break and slow you down when you refactor components. With React Testing Library we're testing the actual DOM and not React-specific implementation details, like ``props`` or ``state``.
```

Comment:
`The main reason to adopt React Testing Library was to avoid testing too much implementation details` -> `We adopted React Testing Library to avoid testing implementation details` (shorter, active voice)
`You want your tests to be maintainable and don't break and slow you down` -> remove multiple "and" clauses, suggest `You want your tests to be maintainable and less susceptible to breaking, which slows you down`